examples/native-demo/src/main/resources/META-INF/native-image/serialization-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "types":[
```

Comment:
empty file might not be necessary?

Reply:
you know that you comment on auto-generated and just copied files, right?